client/components/app-promo/lib/promo-retriever.js

```diff
@@ -0,0 +1,34 @@
+import { sample } from 'lodash';
+
+const promo_options = [
+	{
+		promoCode: 'a0001',
+		message: 'WordPress.com your way — desktop app now available for Mac, Windows, and Linux.'
+	},
+	{
+		promoCode: 'a0002',
+		message: 'Get WordPress.com app for your desktop.'
+	},
+	{
+		promoCode: 'a0003',
+		message: 'WordPress.com app now available for desktop.'
+	},
+	{
+		promoCode: 'a0005',
+		message: 'WordPress.com at your fingertips — download app for desktop.'
+	},
+	{
+		promoCode: 'a0006',
+		message: 'WordPress.com in the palm of your hands — download app for mobile.'
+	}
+];
+
+export const getRandomPromo = ( ) => sample ( promo_options );
```

Comment:
Also shouldn't be an empty space between parens. Wonder if there's an ESLint rule for that...

>No filler spaces in empty constructs (e.g., `{}`, `[]`, `fn()`).

https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#spacing